examples/chip-tool/templates/commands.zapt

```diff
@@ -97,9 +98,20 @@ static void OnInt64sAttributeResponse(void * context, int64_t value)
     command->SetCommandExitStatus(CHIP_NO_ERROR);
 }
 
-static void OnStringAttributeResponse(void * context, const chip::ByteSpan value)
+static void OnOctetStringAttributeResponse(void * context, const chip::ByteSpan value)
```

Comment:
could we place all the 'static' methods that do not depend on code generation inside a separate cpp file that is not generated?

Reply:
Those should be generated at some point with the help of a `partial`.